go.mod

```diff
@@ -32,7 +32,10 @@ require (
 	k8s.io/client-go v0.24.3
 )
 
-require github.com/gorilla/mux v1.8.0
+require (
+	github.com/gorilla/mux v1.8.0
+	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
+)
```

Comment:
Is this change required or something from testing?

Reply:
`go mod tidy` did something strange here. Both dependencies were there before. I have merged them back in to the original list.